.github/workflows/shellcheck.yaml

```diff
@@ -0,0 +1,20 @@
+name: 'Shellcheck'
+
+on:
+  push
```

Comment:
I'm not sure we want shellcheck to run on every push, but I figured this would be a good place to start. If it's too aggressive or runs too frequently, we can scale this back to one of the more targeted events.

Reply:
+1 to your proposed strategy.